lms/djangoapps/ccx/views.py

```diff
@@ -337,6 +342,35 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_date(ccx, node, date_type=None, parent_node=None):
+    """
+    This returns override or master date for section, subsection or a unit.
+
+    :param ccx: ccx instance
+    :param node: chapter, subsection or unit
+    :param date_type: start or due
+    :param parent_node: parent of node
+    :return: start or due date
+    """
+    date = get_override_for_ccx(ccx, node, date_type, None)
+    if date_type == "start":
+        master_date = node.start
+    else:
+        master_date = node.due
+
+    if date is not None:
+        # Setting override date [start or due]
+        date = str(date)[:-9]
```

Comment:
This seems oddly specific, what if the format for dates change? Does this work if dates are localized?